modules/simulator.js

```diff
@@ -63,8 +63,13 @@ function step(ctx, land)
   }
 }
 
+function getRandomInt(min, max) {
```

Comment:
I'd probably put this into a `Helper.js`, since it seems useful/can be used by other classes. Thoughts? @insoochoo @Youngjk 

I think we are randomizing stuff in our code (using `Math.floor([...])`) maybe we can replace it with this method